sdk/rust/oak/src/lib.rs

```diff
@@ -449,6 +449,17 @@ pub fn run_command_loop<N: CommandHandler, R: Iterator<Item = N::Command>>(
     }
 }
 
+/// Trait implemented by structs that correspond to entrypoint of Wasm nodes.
+///
+/// This statically ties together the Wasm entrypoint name (i.e. the name of the Wasm exported
+/// function) to the associated constant of this trait.
+///
+/// This trait is automatically implemented for types used as part of [`entrypoint_command_handler`]
+/// entrypoint definitions.
```

Comment:
I think it would be useful to link to the `entrypoint_node_create` here in the docs, right now it is hard to find where this trait is actually being used.